nodejs-19 was failing with:

```
../../deps/v8/src/heap/cppgc/stats-collector.h:394:48: error: cannot convert '
al::StatsCollector::AllocationObserver*>::iterator' to 'const char*'          
  394 |         std::remove(allocation_observers_.begin(), allocation_observer
                                                                              
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~                     
      |                                                |                      
      |                                                std::vector<cppgc::inte
ocationObserver*>::iterator                                                   
In file included from /usr/include/c++/14/cstdio:42,                          
                 from /usr/include/c++/14/ext/string_conversions.h:45,        
                 from /usr/include/c++/14/bits/basic_string.h:4154,           
                 from /usr/include/c++/14/string:54,                          
                 from /usr/include/c++/14/bits/locale_classes.h:40,           
                 from /usr/include/c++/14/bits/ios_base.h:41,                 
                 from /usr/include/c++/14/ios:44,                             
                 from /usr/include/c++/14/istream:40,                         
                 from /usr/include/c++/14/sstream:40,                         
                 from ../../deps/v8/src/base/logging.h:10,                    
                 from ../../deps/v8/src/base/macros.h:12,                     
                 from ../../deps/v8/src/heap/cppgc/sweeper.h:10:   
...

  158 | extern int remove (const char *__filename) __THROW;                   
      |                    ~~~~~~~~~~~~^~~~~~~~~~                             
In file included from ../../deps/v8/src/heap/cppgc/process-heap-statistics.h:9
                 from ../../deps/v8/src/heap/cppgc/process-heap-statistics.cc:
../../deps/v8/src/heap/cppgc/stats-collector.h: In member function 
     'void cppgcr::ForAllAllocationObservers(Callback)':                                      
../../deps/v8/src/heap/cppgc/stats-collector.h:394:48: error: 
    cannot convert 'al::StatsCollector::AllocationObserver*>::iterator' to 'const char*'          
  394 |         std::remove(allocation_observers_.begin(), allocation_observer
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~                     
      |                                                |                      
      |                                                std::vector<cppgc::inte
rnal::StatsCollector::AllocationObserver*>::iterator          
```

--- a/nodejs-16.yaml
+++ b/nodejs-16.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-16
   version: 16.20.2
-  epoch: 7
+  epoch: 8
   description: "JavaScript runtime built on V8 engine"
   dependencies:
     provides:
@@ -12,6 +12,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.12
 
 environment:
   contents:
@@ -27,9 +30,9 @@ environment:
       - nghttp2-dev
       - openssf-compiler-options
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
@@ -56,7 +59,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

--- a/nodejs-18.yaml
+++ b/nodejs-18.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-18
   version: 18.20.5
-  epoch: 0
+  epoch: 1
   description: "JavaScript runtime built on V8 engine - LTS version"
   copyright:
     - license: MIT
@@ -12,6 +12,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.12
 
 environment:
   contents:
@@ -27,9 +30,9 @@ environment:
       - nghttp2-dev
       - openssf-compiler-options
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
@@ -53,7 +56,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

--- a/nodejs-20.yaml
+++ b/nodejs-20.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-20
   version: 20.18.1
-  epoch: 0
+  epoch: 1
   description: "JavaScript runtime built on V8 engine - LTS version"
   dependencies:
     provides:
@@ -12,6 +12,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.12
 
 environment:
   contents:
@@ -27,9 +30,9 @@ environment:
       - nghttp2-dev
       - openssf-compiler-options
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
@@ -53,7 +56,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

--- a/nodejs-21.yaml
+++ b/nodejs-21.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-21
   version: 21.7.3
-  epoch: 4
+  epoch: 5
   description: "JavaScript runtime built on V8 engine"
   dependencies:
     provides:
@@ -11,6 +11,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.12
 
 environment:
   contents:
@@ -26,9 +29,9 @@ environment:
       - nghttp2-dev
       - openssf-compiler-options
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
@@ -52,7 +55,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

--- a/nodejs-22.yaml
+++ b/nodejs-22.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-22
   version: 22.11.0
-  epoch: 0
+  epoch: 1
   description: "JavaScript runtime built on V8 engine"
   dependencies:
     provides:
@@ -11,6 +11,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.13
 
 environment:
   contents:
@@ -25,12 +28,14 @@ environment:
       - linux-headers
       - nghttp2-dev
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
+  environment:
+    GCC_SPEC_FILE: /dev/null # https://github.com/wolfi-dev/os/issues/34075
 
 pipeline:
   - uses: git-checkout
@@ -51,7 +56,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

--- a/nodejs-23.yaml
+++ b/nodejs-23.yaml
@@ -1,7 +1,7 @@
 package:
   name: nodejs-23
   version: 23.4.0
-  epoch: 0
+  epoch: 1
   description: "JavaScript runtime built on V8 engine"
   dependencies:
     provides:
@@ -11,6 +11,9 @@ package:
   resources:
     cpu: 33
     memory: 64Gi
+
+vars:
+  py-version: 3.13
 
 environment:
   contents:
@@ -25,9 +28,9 @@ environment:
       - linux-headers
       - nghttp2-dev
       - openssl-dev
-      - py3-jinja2
-      - py3-setuptools
-      - python3
+      - py${{vars.py-version}}-jinja2
+      - py${{vars.py-version}}-setuptools
+      - python-${{vars.py-version}}-base
       - samurai
       - wolfi-base
       - zlib-dev
@@ -53,7 +56,7 @@ pipeline:
       export CFLAGS="${CFLAGS/-Os/-O2} $common_flags"
       export CXXFLAGS="${CXXFLAGS/-Os/-O2} $common_flags"
       export CPPFLAGS="${CPPFLAGS/-Os/-O2} $common_flags"
-      python3 configure.py --prefix=/usr \
+      python${{vars.py-version}} configure.py --prefix=/usr \
         --shared-brotli \
         --shared-zlib \
         --shared-openssl \

